react-native/demo/App.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { /*useEffect,*/ useState } from 'react';
+import { ScrollView, View, I18nManager, Text } from 'react-native';
+import { Header, Spacer, wrapIcon } from '@pxblue/react-native-components';
+import { useTheme } from 'react-native-paper';
+import MatIcon from 'react-native-vector-icons/MaterialIcons';
+import { Battery, Heart, Ups, Pie } from '@pxblue/react-native-progress-icons';
+import * as Colors from '@pxblue/colors';
+
+// const backgroundImage = require('./assets/images/farm.jpg');
+const MenuIcon = wrapIcon({ IconClass: MatIcon, name: 'menu', flip: I18nManager.isRTL });
+
+type LabelPosition = undefined | 'top' | 'right' | 'bottom' | 'left';
+const positions: LabelPosition[] = [undefined, 'left', 'top', 'right', 'bottom'];
+
+export const App: React.FC = () => {
+    const theme = useTheme();
+    const [percent /*, setPercent*/] = useState(50);
```

Comment:
If we're not changing this value, should this just be a number const instead of using the state hook?

Reply:
I left in the logic for looping the icons through the percent from 0-100 (see also the commented useEffect hook).

I commented that out because it is a big performance hog when all of the icons are up on the screen, but didn't want to take it all the way out in case somebody wants to play with that in the future.